includes/data-stores/abstract-wc-order-data-store-cpt.php

```diff
@@ -102,10 +103,11 @@ public function read( &$order ) {
 
 		$order->set_props(
 			array(
-				'parent_id'     => $post_object->post_parent,
-				'date_created'  => 0 < $post_object->post_date_gmt ? wc_string_to_timestamp( $post_object->post_date_gmt ) : null,
-				'date_modified' => 0 < $post_object->post_modified_gmt ? wc_string_to_timestamp( $post_object->post_modified_gmt ) : null,
-				'status'        => $post_object->post_status,
+				'parent_id'          => $post_object->post_parent,
+				'date_created'       => 0 < $post_object->post_date_gmt ? wc_string_to_timestamp( $post_object->post_date_gmt ) : null,
+				'date_modified'      => 0 < $post_object->post_modified_gmt ? wc_string_to_timestamp( $post_object->post_modified_gmt ) : null,
+				'status'             => $post_object->post_status,
+				'prices_include_tax' => 'yes' === get_option( 'woocommerce_prices_include_tax' ),
```

Comment:
Sorry, I still don't see how setting `prices_include_tax` on line 110 solves anything when [on line 114 immediately after this, we're already setting `prices_include_tax`](https://github.com/woocommerce/woocommerce/blob/65cfd892422d9ea6a7725a61b942fe39b038d99d/includes/data-stores/abstract-wc-order-data-store-cpt.php#L256). Have I missed something?

Reply:
Dunno must have been glitch at time. I see orders are forced to exist now shrug